environments/kolla/files/overlays/ceilometer/event_pipeline.yaml

```diff
@@ -0,0 +1,12 @@
+---
+sources:
+    - name: event_source # 'source name'
+      events:
+          - "*" # 'event filter'
+      sinks:
+          - event_sink # 'sink name'
+sinks:
+    - name: event_sink # 'sink name'
+      publishers: # 'list of publishers'
```

Comment:
if we have inline comments, these should provide additional context.

these comments currently only replicate the yaml structure, e.g.

"sources -> name" becomes "source name" which does not yield any additional useful information, like what source is, or why does it need a name.

so in it's current form, these comments should imho all get removed.

Reply:
Thanks for the hint. I put more context to these comments.

Reply:
@artificial-intelligence Do you think this request is resolved?